debuggers/gdb/gdb_mi_driver.py

```diff
@@ -29,57 +29,100 @@ def __init__(self, base_args, base_script_file_path, regression_args, regression
         self.run_parallel_raw_command("source " + os.path.join(dirname, "gdb_commands.py"))
 
     def run_parallel_command(self, command):
-        base_response = self.run_single_command(command, "base")
-        regressed_response = self.run_single_command(command, "regressed")
+        # start both execution in parallel
+        self.base_gdb_instance.send(((" {command}\n".format(command = command),), {"timeout_sec": 60}))
+        self.regressed_gdb_instance.send(((" {command}\n".format(command = command),), {"timeout_sec": 60}))
+        
+        # wait till base is done
+        raw_result = self.base_gdb_instance.recv()
+        
+        # make sure all output is flushed
+        # time.sleep(.005)
+        self.base_gdb_instance.send((("",), {"timeout_sec": 60}))
+        raw_result += self.base_gdb_instance.recv()
```

Comment:
Do we need this?
I observed that we do not have this in the `run_single_special_command` function.

Reply:
I think this is a remaining of the previous efforts to capture GDB response where we listened for stdout output. I think this is no longer needed since we are using the other approach based on the IoManager.